RMSD.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os, sys, glob
+import openeye.oechem as oechem
+import openmoltools
```

Comment:
same thing with openmoltools, try to get in the habit of only importing modules you actually use.